torchtext/experimental/models/xlmr_transform.py

```diff
@@ -0,0 +1,28 @@
+import os
+import torch.nn as nn
+from torchtext.experimental.transforms import sentencepiece_tokenizer
+from torchtext.experimental.vocab import load_vocab_from_file
+from typing import List
+
+
+class XLMRTransform(nn.Module):
+    """XLM-R encode transform."""
+
+    def __init__(self, tokenizer, vocab):
+        super(XLMRTransform, self).__init__()
+        self.tokenizer = tokenizer
+        self.vocab = vocab
+
+    @classmethod
+    def from_pretrained(cls, directory='./',
```

Comment:
Could you please comment on why we might want to keep a `directory` here?

Reply:
I didn't fix this one here. Good catch.